app.js

```diff
@@ -1,16 +1,26 @@
 const express = require("express");
 const app = express();
 const { getCategories } = require("./controllers/categories.controller");
+const { getReviewId } = require("./controllers/reviews.controller");
 
 module.exports = app;
 
 app.get("/api/categories", getCategories);
 
+app.get("/api/reviews/:review_id", getReviewId);
+
 app.all("*", (req, res) => {
   res.status(404).send({ msg: "request not found" });
 });
 
 app.use((err, req, res, next) => {
-  console.log(err);
-  res.status(500).send("Server Error!");
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
```

Comment:
Use the notes from the error handling sent out last week to see if this can be refactored into something tidier.